src/components/Coin.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import '../styles/Coin.css';
+
+const CoinComponent = ({ name, image, price }) => (
+  <div className="coin">
+    <img src={image} alt={name} className="coin-image" />
+    <h1 className="coin-name">{name}</h1>
+    <p className="coin-price">
+      $
+      {price}
+    </p>
+  </div>
+);
```

Comment:
Nice job making the layout two columns as indicated by the previous reviewer. However, there are other points that was requested by the reviewer but you seem to have missed them. Kindly check below and update your project accordingly :pray: 

- [ ] Each card have different background color intensity (alternating between dark and light version of the same color) as shown on the image below. (check numbers 1,2,3,4).
- [ ] You seem to use more than one font color (white, black, blue, etc) which is not suppose to be so. The design expects you to use only one font-color throughout the project.
- [ ] Kindly add the missing arrow which indicates the cards are clickable and should navigate the user to the correct page to improve the user experience of your app.

### Your design 

![Screenshot from 2023-08-26 16-20-53](https://github.com/Peter-Kibet/metrics-app-react-capstone/assets/65233860/dd93ac7a-a84e-49cc-ae73-fef7a3c1a127)



### Actual design 


![Screenshot from 2023-08-26 16-09-26](https://github.com/Peter-Kibet/metrics-app-react-capstone/assets/65233860/70bb8f64-8562-4d58-8ceb-4f59e5f74431)